pkg/reconciler/sugar/namespace/namespace_test.go

```diff
@@ -64,12 +67,39 @@ func (t *testConfigStore) ToContext(ctx context.Context) context.Context {
 }
 
 func TestEnabled(t *testing.T) {
+	// Context with DefaultConfig
+	c := config.Config{
+		Defaults: &config.Defaults{
+			ClusterDefault: &config.ClassAndBrokerConfig{
+				BrokerClass: "AValidBrokerClass",
+			},
+		},
+	}
+
+	ctx := config.ToContext(context.Background(), &c)
+
 	// Events
 	brokerEvent := Eventf(corev1.EventTypeNormal, "BrokerCreated", "Default eventing.knative.dev Broker created.")
 
 	// Objects
 	broker := resources.MakeBroker(testNS, resources.DefaultBrokerName)
 
+	// Reactors
+	defaultFunc := func(action clientgotesting.Action) (handled bool, ret runtime.Object, err error) {
+		if verb := action.GetVerb(); verb != "create" {
+			return false, nil, nil
+		}
+
+		got := action.(clientgotesting.CreateAction).GetObject()
+		obj, ok := got.(apis.Defaultable)
+		if !ok {
+			return false, nil, nil
+		}
+
+		obj.SetDefaults(ctx)
+		return false, got, nil
+	}
```

Comment:
Can we extract this function to a common one and use it in both tests?

Reply:
Yep 👍 done